src/SourceGenerators/SourceGeneratorHelpers/Helpers/PlatformHelper.cs

```diff
@@ -15,34 +15,18 @@ public class PlatformHelper
 	{
 		public static bool IsValidPlatform(GeneratorExecutionContext context)
 		{
-			var evaluatedValue = context.GetMSBuildPropertyValue("TargetPlatformIdentifier");
-			var windowsAppSDKWinUI = context.GetMSBuildPropertyValue("WindowsAppSDKWinUI");
 			var projectTypeGuids = context.GetMSBuildPropertyValue("ProjectTypeGuidsProperty");
 
-			var isUAP = evaluatedValue?.Equals("UAP", StringComparison.OrdinalIgnoreCase) ?? false;
-
-			// Validate if we're running in a valid context, where CompileVisibleProperties for
-			// Uno's generators have been included. This may not be the case when building UAP targets
-			// for libraries, during the XamlPreCompile target.
-			var isInvalid =
-				string.IsNullOrEmpty(context.GetMSBuildPropertyValue("Configuration"))
-				&& string.IsNullOrEmpty(context.GetMSBuildPropertyValue("AssemblyName"))
-				&& string.IsNullOrEmpty(context.GetMSBuildPropertyValue("RootNamespace"));
-
 			// Those two checks are now required since VS 16.9 which enables source generators by default
 			// and the uno targets files are not present for uap targets.
 			var isWindowsRuntimeApplicationOutput = context.Compilation.Options.OutputKind == OutputKind.WindowsRuntimeApplication;
 			var isWindowsRuntimeMetadataOutput = context.Compilation.Options.OutputKind == OutputKind.WindowsRuntimeMetadata;
 
-			var isWinAppSDK = windowsAppSDKWinUI?.Equals("True", StringComparison.OrdinalIgnoreCase) ?? false;
```

Comment:
Generator is removed when `WindowsAppSDKWinUI` is true.

So `isWinAppSDK` is always false. Hence, removed the redundant check.